status/k8s.go

```diff
@@ -354,5 +354,23 @@ func (k *K8sStatusCollector) status(ctx context.Context) (*Status, error) {
 		}
 	}
 
+	disabled, err = k.deploymentStatus(ctx, status, defaults.HubbleUIServiceName)
```

Comment:
I think this (and the following) should use `defaults.HubbleUIDeploymentName` instead of `defaults.HubbleUIServiceName`. It work as-is because they are both "hubble-ui", but `k.deploymentStatus` expect a Deployment.

Reply:
Oh, my bad. I will change it right away.

Reply:
Please don't forget to squash your commits once you're done!

Reply:
Okay, I have a bad experience doing it but I will do my research and do it correctly 👍🏻